.gitlab-ci.yml

```
@@ -0,0 +1,44 @@
+stages:
+  - build_and_test
+  - push_to_dockerhub
+  - deploy_on_EC2
+
+
+build_job:
+  stage: build_and_test
+  script: 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)